templates.ts

```diff
@@ -64,6 +64,9 @@ export function generateRendererFile(viewHash: string, viewFunction: string, dec
     } else {
         initBody = initBodyWithoutDecoder(viewHash, viewFunction);
     }
+    let newLinesStr = newLines !== undefined ? (newLines.toString().charAt(0).toUpperCase() + newLines.toString().slice(1)) : "True";

```

Comment:
this is really hard to read, and i'm not sure why it's so complicated?

Reply:
it's just checking to see if newLines was specified, and if it was then capitalizes the first letter cause bools in elm start with an uppercase letter

Reply:
I would just make it like this: `newLineStr = newLines ? "True" : "False"`. Much simpler :)

Reply:
it doesn't work unless newLines/indent is explicitly checked against undefined

Reply:
Right, but I mean make it just "True" or "False". Don't do this complicated approach of modifying the string -- go for the simplest approach.